### PR TITLE
Grouped RevAPI updates

### PR DESCRIPTION
**Pull request by cescoffier**
- bump revapi-maven-plugin to version 0.15.0
- bump revapi-java to version 0.28.0
- bump revapi-reporter-json and revapi-reporter-text to version 0.15.0

### CODE REVIEW

**Comment by cescoffier** [comment]
@ozangunalp I WON \o/

**Review by ozangunalp — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -525,9 +525,17 @@
               <configuration>
                 <versionFormat>^\d+\.\d+\.\d+$</versionFormat>
                 <failSeverity>breaking</failSeverity>
+                <checkDependencies>false</checkDependencies>
                 <analysisConfigurationFiles>
                   <file>${project.basedir}/revapi.json</file>
                 </analysisConfigurationFiles>
+                <analysisConfiguration>
+                  <revapi.java>
+                    <missing-classes>
```
Comment (ozangunalp) [inline pom.xml:534]: Wasn't this already active or a new rule?

Comment (cescoffier) [reply pom.xml:534]: Let me try to remove it - that was an approach to fix class loading errors (which do not happen with 17).

**Comment by ozangunalp** [comment]
I tihnk added Kafka breaking changes were from the previous version (at least first 2, I don't know it didn't show me others before)

**Review by ozangunalp — APPROVED** [review]

**Pull request merged**